usage/index.rst

```diff
@@ -506,11 +506,25 @@ Jobs are scheduled subject to Slurm's `Multifactor Priority Plugin <https://slur
 Grace-Hopper Pilot
 ------------------
 
-Bede contains 1 NVIDIA Grace-CPU node and 7 NVIDIA Grace-Hopper nodes.
+Bede contains 1 NVIDIA Grace-CPU node and 8 NVIDIA Grace-Hopper nodes.
 
 Each Grace-CPU node node contains a single `Grace Superchip <https://www.nvidia.com/en-gb/data-center/grace-cpu/>`_, containing two 72-core 64-bit ARM CPUs. Further details are listed on the :ref:`hardware` page.
 
-Each Grace-Hopper node contains a single `Grace Hopper Superchip <https://www.nvidia.com/en-gb/data-center/grace-hopper-superchip/>`_, containing one 72-core 64-bit ARM CPU and one 96GB Hopper GPU with NVLink-C2C providing 900GB/s of bidirectional bandwidth between the CPU and GPU. Further details are listed on the :ref:`hardware` page.
+There are two types of Grace-Hopper node. 7 nodes contain:
+
+- 1x `Grace Hopper Superchip <https://www.nvidia.com/en-gb/data-center/grace-hopper-superchip/>`_
+
+  - one 72-core 64-bit ARM CPU
+  - one 96GB Hopper GPU
+  - NVLink-C2C providing 900GB/s of bidirectional bandwidth between the CPU and GPU.
+
```

Comment:
Potentially missing an extra sentence here to match 

> There are two types of Grace-Hopper node. 7 nodes contain:


```suggestion

1 node contains:

```

Or the first sentence could be changed and the `7` and `1` moved into the bullets somehow?

Reply:
Looks good